lib/dradis/plugins/templates.rb

```diff
@@ -28,7 +46,32 @@ def copy_templates(args={})
           end
         end
 
-        def plugin_templates(args={})
+        def migrate_templates_to_mappings(args = {})
+          # return if the integration doesn't provide any templates ex. projects, cve
+          return unless paths['dradis/templates'].existent.any?
+          @integration_name = plugin_name.to_s
+          # return if templates have already been migrated (mappings exist for the integration)
+          return if Mapping.where(component: @integration_name).any?
+
+          templates_dir = args.fetch(:from)
+          @integration_templates_dir = File.join(templates_dir, @integration_name)
+
+          if uploaders.count > 1
+            migrate_multiple_uploaders(@integration_name)
+          else
+            template_files = Dir["#{@integration_templates_dir}/*.template"]
+            return unless template_files.any?
+
+            template_files.each do |template_file|
+              next unless File.exist?(template_file)
```

Comment:
Can you use `#existent` again in L62 to avoid this check?

Reply:
I can't use the same `paths` here since I'm looking in the instance's `/templates` dir, but `/existent` can only be used on a `Rails::Paths::Path` object. Seems like a 🐰🕳 at this point but worth revisiting